editor/editor_inspector.cpp

```diff
@@ -1216,39 +1225,78 @@ void EditorInspectorSection::_notification(int p_what) {
+			}
+
+			int header_width = get_size().width - section_indent;
+			int header_offset_x = 0.0;
+			if (!rtl) {
+				header_offset_x += section_indent;
+			}
+
+			// Draw header area.
+			Rect2 header_rect = Rect2(Vector2(header_offset_x, 0.0), Vector2(header_width, header_height));
 			Color c = bg_color;
 			c.a *= 0.4;
 			if (foldable && header_rect.has_point(get_local_mouse_position())) {
 				c = c.lightened(Input::get_singleton()->is_mouse_button_pressed(MouseButton::LEFT) ? -0.05 : 0.2);
 			}
 			draw_rect(header_rect, c);
 
+			// Draw header title and folding arrow.
 			const int arrow_margin = 2;
 			const int arrow_width = arrow.is_valid() ? arrow->get_width() : 0;
 			Color color = get_theme_color(SNAME("font_color"));
-			float text_width = get_size().width - Math::round(arrow_width + arrow_margin * EDSCALE);
-			draw_string(font, Point2(rtl ? 0 : Math::round(arrow_width + arrow_margin * EDSCALE), font->get_ascent(font_size) + (header_height - font->get_height(font_size)) / 2).floor(), label, rtl ? HORIZONTAL_ALIGNMENT_RIGHT : HORIZONTAL_ALIGNMENT_LEFT, text_width, font_size, color);
+			float text_width = get_size().width - Math::round(arrow_width + arrow_margin * EDSCALE) - section_indent;
+			Point2 text_offset = Point2(0, font->get_ascent(font_size) + (header_height - font->get_height(font_size)) / 2);
+			HorizontalAlignment text_align = HORIZONTAL_ALIGNMENT_LEFT;
+			if (rtl) {
+				text_align = HORIZONTAL_ALIGNMENT_RIGHT;
+			} else {
+				text_offset.x = section_indent + Math::round(arrow_width + arrow_margin * EDSCALE);
+			}
+			draw_string(font, text_offset.floor(), label, text_align, text_width, font_size, color);
 
 			if (arrow.is_valid()) {
+				Point2 arrow_position = Point2(0, (header_height - arrow->get_height()) / 2);
 				if (rtl) {
-					draw_texture(arrow, Point2(get_size().width - arrow->get_width() - Math::round(arrow_margin * EDSCALE), (header_height - arrow->get_height()) / 2).floor());
+					arrow_position.x = get_size().width - section_indent - arrow->get_width() - Math::round(arrow_margin * EDSCALE);
 				} else {
-					draw_texture(arrow, Point2(Math::round(arrow_margin * EDSCALE), (header_height - arrow->get_height()) / 2).floor());
+					arrow_position.x = section_indent + Math::round(arrow_margin * EDSCALE);
 				}
+				draw_texture(arrow, arrow_position.floor());
 			}
 
+			// Draw dropping highlight.
 			if (dropping && !vbox->is_visible_in_tree()) {
 				Color accent_color = get_theme_color(SNAME("accent_color"), SNAME("Editor"));
 				draw_rect(Rect2(Point2(), get_size()), accent_color, false);
 			}
+
+			// Draw section indentation.
+			if (section_indent_style.is_valid() && section_indent > 0) {
+				Rect2 indent_rect = Rect2(Vector2(), Vector2(indent_depth * section_indent_size, get_size().height));
+				if (rtl) {
+					indent_rect.position.x = get_size().width - section_indent + section_indent_style->get_margin(SIDE_RIGHT);
```

Comment:
Probably should be:
```suggestion
					indent_rect.position.x = get_size().width - indent_rect.size.x - section_indent_style->get_margin(SIDE_RIGHT);
```

Reply:
I think it should be the left margin subtracted then to get the same result. But either way, the value should be correct, so I can change it, but only if you think it makes the logic clearer.

Reply:
I double-checked using a noticeably big number for a margin, and the behavior seems to be correct:

LTR
![godot windows tools 64_2022-02-10_20-27-19](https://user-images.githubusercontent.com/11782833/153462766-42bf9818-04f5-47b9-90f2-8e2f4c7f5ce9.png)

RTL
![godot windows tools 64_2022-02-10_20-26-49](https://user-images.githubusercontent.com/11782833/153462772-1c8a70be-29eb-4198-b824-50edafd4d7c3.png)

So I'll leave it as is for now.

Reply:
It's not really obvious without checking what `section_indent` is, but it seems to be the same, should be OK as is.